src/Nordea/Components/FileUpload.elm

```diff
@@ -396,6 +395,15 @@ onSelectFiles msg =
     Events.on "change" filesDecoder
 
 
+strings :
+    { uploadDescription1 : Translation
+    , uploadDescription2 : Translation
+    , uploadDescription3 : Translation
+    , acceptedFileTypes : Translation
+    , and : Translation
+    , dropToUploadFile : Translation
+    , uploadedFiles : Translation
+    }
```

Comment:
I'm a little skeptical about the value of adding type annotations to the translations record. It's not too bad here in the component library I guess, but in PartnerHub we have many more translations per file. My concern is that it adds mostly noise and not much value, since it's a hardcoded structure and Elm has "perfect" type inference.

What are your thoughts about it?

Reply:
I kinda agree. I think I just did it to get one less warning.
Should I just remove this annotation, or remove the whole Translation type?

Reply:
I think we can keep the translation type, and just remove the annotation. Is the warning from elm-review? Maybe we can add an exception for the translations record?

Reply:
Seems like just some stuff in vs-code that complained. We can just ignore that.
Elm-review does not complain.

I removed it now 👍